README.md

```diff
@@ -109,14 +109,15 @@ Star :star:  the repo to help the developers :innocent:
 3) cd (change directory) in to the project folder
 4) Run `npm install` in your terminal
 5) Run `ng serve` to run the Angular frontend
-6) Run `npm run start:server` to run the backend Node server
-7) Open your browser and navigate to `http://localhost:4200/`
+6) Start backend database using docker. Run `docker run --name rems_local_pharmacy-information-system-database --expose 27017 -p 27017:27017 -e MONGO_INITDB_ROOT_USERNAME='pharmacy-information-root' -e MONGO_INITDB_ROOT_PASSWORD-'pharmacy-information-password' -v rems_local_pharmacy-infomation-system-database:/data/db mongo`
```

Comment:
when I try to run this I get

`error: missing 'MONGO_INITDB_ROOT_USERNAME' or 'MONGO_INITDB_ROOT_PASSWORD'
       both must be specified for a user to be created`

Any ideas?

Reply:
ahh there was a typo:  `-e MONGO_INITDB_ROOT_PASSWORD-'pharmacy-information-password' ` is supposed to be  `-e MONGO_INITDB_ROOT_PASSWORD='pharmacy-information-password' `. I'll update the readme